src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenCheckedTests.cs

```diff
@@ -383,7 +383,7 @@ public static long Sub_Ptr(C* a, C* b)
     }
 }
 ";
-            var verifier = CompileAndVerify(source, options: TestOptions.UnsafeReleaseDll);
+            var verifier = CompileAndVerify(source, options: TestOptions.UnsafeReleaseDll, verify: Verification.Fails);
```

Comment:
:bulb: If you do happen to know the reason why verification is skipped in the cases where it is, it would be helpful to add a comment. It appears to be a minority of cases overall (i.e. hopefully not too much work to figure out the reason).

Edit: This suggestion may not be sustainable now that I've seen how many times `Fails` is used below.

Reply:
We shoudl eventually not have or have very few "Skips" - The reasons for "skipped" are typically bugs or some functional deficiency in PEVerify - i.e different behavior in 32bit and 64bit (since the actual algo is coupled with JIT and the implementations are different), or major difficulties with net modules.

Most of the Skips were already there, so I did not go and explain them. I have explained couple that I had to add.

Once we switch to ILVerify, we might need to add more "Skip" cases - since there will be bugs :-) and then work on reducing them to the minimum.